products/jenkins.md

```diff
@@ -27,8 +27,8 @@ releases:
     releaseLabel: "Regular"
     releaseDate: 2016-04-20
     eol: false
-    latest: "2.402"
-    latestReleaseDate: 2023-04-25
+    latest: "2.404"
```

Comment:
Strange, 2.403 was not catched despite being released last week. Maybe the tag was not pushed ?